client/components/student/student.service.js

```diff
@@ -7,8 +7,7 @@ app.factory('Student', function($resource) {
     id: '@_id'
   }, {
     list: {
-      method: 'GET',
-      isArray: true
+      method: 'GET'
```

Comment:
I don't think this change makes sense. This custom resource method should probably be deleted, but this will change the "list" which returns an array into a "get" which returns a single object. Instead of using this "list" the built in `Student.get({id: $stateParams.id})` should be used. 

However, as stated in the above comment. I don't think the `.list` call is needed for this change..

However, none of this is needed because